docs/bsp.md

```diff
@@ -913,6 +913,33 @@ during compilation before completing the response.
 The client will get a `originId` field in `RunResult` if the `originId` field in the
 `RunParams` is defined.
 
+### Clean Cache Request
+
+The clean cache request is sent from the client to the server to remove cached artifacts that are associated with a given build target.
```

Comment:
I think some clarification of semantics would be good here, with the acknowledgement that they may be tool-specific.
For example:
After a successful cleanCache request, it is expected that disk artifacts created by tasks on the target have been removed, and compile requests to the given targets will perform a full recompile. The build tool defines the exact semantics.

Reply:
Good point, done.